pkg/accounting/tally/tally.go

```diff
@@ -153,18 +153,26 @@ func (t *tally) queryBW(ctx context.Context) error {
 	}
 
 	// sum totals by node id ... todo: add nodeid as SQL column so DB can do this?
-	bwTotals := make(map[string]int64)
+	var bwTotals accounting.BWTally
+	for i := range bwTotals {
+		bwTotals[i] = make(map[string]int64)
+	}
 	var latestBwa time.Time
 	for _, baRow := range bwAgreements {
 		rbad := &pb.RenterBandwidthAllocation_Data{}
 		if err := proto.Unmarshal(baRow.Agreement, rbad); err != nil {
 			t.logger.DPanic("Could not deserialize renter bwa in tally query")
 			continue
 		}
+		pbad := &pb.PayerBandwidthAllocation_Data{}
+		if err := proto.Unmarshal(rbad.GetPayerAllocation().GetData(), pbad); err != nil {
```

Comment:
is it possible `rbad.GetPayerAllocation()` will return nil ?

Reply:
I'm not sure.  I think we shouldn't be storing agreements without payers.  I would hope we catch the kinds of things on communication edges, rather than checking continuously throughout our code.